pkg/apis/kops/cluster.go

```diff
@@ -207,6 +205,17 @@ type ClusterSpec struct {
 	ClusterAutoscaler *ClusterAutoscalerConfig `json:"clusterAutoscaler,omitempty"`
 	// WarmPool defines the default warm pool settings for instance groups (AWS only).
 	WarmPool *WarmPoolSpec `json:"warmPool,omitempty"`
+
+	// ServiceAccountIssuerDiscovery configures the OIDC Issuer for ServiceAccounts.
+	ServiceAccountIssuerDiscovery *OIDCIssuerConfig `json:"serviceAccountIssuerDiscovery,omitempty"`
```

Comment:
I think the type should have the field name as a prefix.

Reply:
I was wondering if we'd use this type for other properties too. But I guess not. I'll amend.